mbq/tokens/decoder.py

```diff
@@ -6,6 +8,68 @@
 from . import exceptions
 
 
+class Token:
+    """
+    Convenience class for easy access to the fields of a JWT.
+
+    All properties use the full name of the claims specified by the
+    JWT RFC[1]. Additionally, `expires_at` is exposed as an alias to
+    `expiration_time` for consistency.
+
+    The `dict` read-only interface (`__getitem__` and `get`) are
+    provided for backwards-compatibility only; new code should use
+    the properties.
+
+    [1]: https://tools.ietf.org/html/rfc7519#section-4.1
+    """
+    def __init__(self, raw, decoded):
+        self.raw = raw
+        self._decoded = decoded
+
+    def __getitem__(self, key):
+        return self._decoded[key]
+
+    def get(self, key, default=None):
+        return self._decoded.get(key, default)
+
+    @property
+    def issuer(self):
+        return self._decoded.get('iss')
+
+    @property
+    def subject(self):
+        return self._decoded.get('sub')
+
+    @property
+    def audience(self):
+        return self._decoded.get('aud')
+
+    @property
+    def expiration_time(self):
+        ts = self._decoded.get('exp')
+        if ts is None:
+            return
+        return dt.datetime.fromtimestamp(ts, dt.timezone.utc)
+
+    @property
+    def expires_at(self):
```

Comment:
I'm not sure it's a good idea to provide two ways of grabbing the token expiration but `expiration_time` feels inconsistent with `issued_at` and `not_before` and we tend to not use the word "time" in our code.

I think we should pick one: go with the standard `expiration_time` or go with what seems right for us with `expires_at`.

Reply:
I think using `expires_at` makes sense; if we go with that, could we leave a comment explaining that rationale?